### PR TITLE
accept function for 'focused' prop

### PR DESCRIPTION
**Pull request by JohnC-80**
By default, HotKeys only executes handlers that are within its children's outermost DOM element (either the FocusTrap wrapping div or the outer div of the children.) The `focused` prop is a boolean which simplifies the execution of handlers under certain situations by allowing the DOM element to be considered in focus regardless of where the actual focus is. Adding this prop as a function that will allow the handler to check at the exact time when it's handled whether the focus criteria is met is another option over an extra property somewhere in the state.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 /node_modules/
+/umd/
+/es/
```
Comment (aliaksei-chumakou) [inline .gitignore:3]: is it by intention? Because `index.es.js` relies on `require()` from that dir

Comment (zburke) [reply .gitignore:3]: I believe those directories are generated by the build process.

Comment (rasmuswoelk) [reply .gitignore:3]: I'm getting this error when running the latest version of stripes-react-hotkeys:

image

Running `stripes serve` in my `stripes-sample-platform`-folder. I'm on stripes v.1.6.1000131

Am I missing a build step or something? 🤔

Comment (rasmuswoelk) [reply .gitignore:3]: Running `yarn prepublish` in the directory worked.

Comment (rasmuswoelk) [reply .gitignore:3]: You will only get this error if you pulled down the module into your workspace for development and you didn't run the above command.

Comment (rasmuswoelk) [reply .gitignore:3]: This is a yarn workspace issue: https://github.com/yarnpkg/yarn/issues/3911

**Pull request merged**